src/ontology/edits/oeo-physical.omn

```diff
@@ -814,6 +814,9 @@ Class: <http://purl.obolibrary.org/obo/BFO_0000023>
     
 Class: <http://purl.obolibrary.org/obo/BFO_0000024>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A tangential proper part is a fiat object part that conforms to the qualitative spatially representaion of TPP in RCC8."@de
```

Comment:
Here should be the definition/elucidation of `fiat object part` and not of `tangential proper part`